101-app-function/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+  "itemDisplayName": "Function App.",
+  "description": "This template deploy a Function App. Also creates a Service Plan and a Storage Account.",
+  "summary": "This template deploy a Function App.",
```

Comment:
This template deploys an empty function app and a hosting plan.

Reply:
Thanks